extension/src/frequency.rs

```diff
@@ -399,6 +399,92 @@ pub mod toolkit_experimental {
     }
 
     ron_inout_funcs!(SpaceSavingAggregate);
+
+    pg_type! {
+        #[derive(Debug)]
+        struct SpaceSavingBigIntAggregate<'input> {
+            num_values: u32,
+            topn: u32,
+            values_seen: u64,
+            freq_param: f64,
+            counts: [u64; self.num_values], // JOSH TODO look at AoS instead of SoA at some point
+            overcounts: [u64; self.num_values],
+            datums: [i64; self.num_values],
```

Comment:
I believe something like this will work now
```rust
pg_type! {
        #[derive(Debug)]
        struct SpaceSavingBigIntAggregate<'input> {
            num_values: u32,
            topn: u32,
            values_seen: u64,
            freq_param: f64,
            elements: [Entry; self.num_values],
    }
}


flat_serialize! {
    #[derive(Debug)]
    struct Entry {
        count: u64,
        overcount: u64,
        datum: i64,
    }
}
```
Though I can't say if it would be a good idea